week-1.md

```diff
@@ -0,0 +1,83 @@
+
+So, personally, my recommendation is to focus almost all of your review effort on understanding the proposal and finding ways to make it work better, and not about whether you'll ultimately end up "for" or "against" it. That way your review effort will make things better whatever ends up happening. That doesn't mean you shouldn't form an opinion as to whether the change is a good one or not, of course, just try not to make that your primary focus!
+
+## Taproot
+
+The current draft of the taproot BIP can be found at:
+
+ * https://github.com/sipa/bips/blob/bip-schnorr/bip-taproot.mediawiki
+
+Remember that this is being actively worked on, so there may have been changes if you've read it previously!
+
+We'll be spending more time later on how "script path" spends work and the details of how signatures work, so you may wish to skip or skim the sections titled "script path spending", "Signature validation rules", and "Constructing and spending Taproot outputs" for now.
+
+Ideally reading the BIP (and its references) will already make it clear what the purpose of the proposal is; and how it achieves that purpose. Some good general questions to think about might include:
+
+ * Do the goals make sense?
+ * Do the design decisions follow from the goals?
+ * What are the tradeoffs, and do they make sense?
+ * How will this affect people who use Bitcoin?
+ * Are there better ways to achieve these goals?
+ * Do all the details of the proposal fit together and work?
+
+You may want to review the sample implementation, at
+
+ * https://github.com/sipa/bitcoin/commits/taproot
+
+or you may want to review past discussion, such as on the bitcoin-dev mailing list, in the OpTech newsletters, or looking through the git changelog of the BIP draft or the PRs or issues in the repository. If you do want to explore some of these paths, there are links in the curriculum doc mentioned above. If you find more resources, you may wish to open a PR to add them to the Resources doc:
+
+ * https://github.com/ajtowns/taproot-review/blob/master/Resources.md
+
+Alternatively, you may want to just focus on the BIP text directly to see if it makes sense on its own.
+
+If there are parts you don't understand discuss them with others (such as your study group, or asking at the Q&A sessions, or raising a question on bitcoin.stackexchange.org, or however else you like. If that resolves the confusion, great! If not, it might be worth creating an issue or pull request against the bip-schnorr draft on github. It can be hard to find a balance between getting the details right and caught up in bikeshedding, so talking to other people first is usually a good idea!
```

Comment:
Nit: place a comma between "understand" and "discuss".  Also this paragraph has an unclosed parenthesis.